src/app/examples/form/components/form.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, OnInit, ViewChild, OnDestroy } from '@angular/core';
+import {
+  Validators,
+  FormBuilder,
+  AbstractControl,
+  FormGroup,
+  FormControl
+} from '@angular/forms';
+import { CdkTextareaAutosize } from '@angular/cdk/text-field';
+import { ROUTE_ANIMATIONS_ELEMENTS } from '@app/core';
+import { Store, select } from '@ngrx/store';
+import { Subject } from 'rxjs';
+import { State } from '@app/examples/examples.state';
+import { takeUntil } from 'rxjs/operators';
+import { selectForm } from '@app/examples/form/form.selectors';
+import { ActionFormRetrieve } from '@app/examples/form/form.actions';
+import { Form } from '@app/examples/form/form.model';
+import { MatSnackBar } from '@angular/material';
```

Comment:
Please, have a look into imports. I saw many unused imports in some of the changed files. Also it would be nice to follow ordering as in other files:

1. library imports (`@angular`, `@ngrx`, `rxjs`..)
2. app wide imports (`@app`, `@env`, ...)
3. module relative imports (`../form.actions`, ...)

and please add space between the groups.

 You can see it in action in todos module for example....